reflex/components/component.py

```diff
@@ -1643,10 +1643,11 @@ def compile_from(cls, component: BaseComponent) -> BaseComponent:
         Returns:
             The memoized component tree.
         """
+        from reflex.components.forms.input import InputGroup
         from reflex.components.layout.foreach import Foreach
 
         # Foreach must be memoized as a single component to retain index Var context.
-        if not isinstance(component, Foreach):
+        if not isinstance(component, (Foreach, InputGroup)):
```

Comment:
We should update the comment here too. Do we need to add any other components to this tuple?